orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/QueueExecutionRunner.kt

```diff
@@ -16,13 +16,16 @@
 
 package com.netflix.spinnaker.orca.q
 
+import com.netflix.spinnaker.orca.api.pipeline.ExecutionRunner
+import com.netflix.spinnaker.orca.api.pipeline.models.ExecutionEngineVersion
 import com.netflix.spinnaker.orca.api.pipeline.models.PipelineExecution
-import com.netflix.spinnaker.orca.pipeline.ExecutionRunner
+import com.netflix.spinnaker.orca.api.pipeline.models.ExecutionEngine.v3
 import com.netflix.spinnaker.q.Queue
 import com.netflix.spinnaker.security.AuthenticatedRequest
 import org.springframework.stereotype.Component
 
 @Component
```

Comment:
Just curious. We drive the decision of what runner to use based on the annotation being present ..?

Reply:
Yeah, I could change that if we don't like it but it seemed to make sense to me.  I didn't want to add a field to `ExecutionRunner` since the composite `ExecutionEngineRunner` takes a list of `ExecutionRunner` objects.